src/Sylius/Bundle/ApiBundle/Serializer/ProductVariantNormalizer.php

```diff
@@ -64,16 +64,21 @@ public function normalize($object, $format = null, array $context = [])
         $context[self::ALREADY_CALLED] = true;
 
         $data = $this->normalizer->normalize($object, $format, $context);
+        $channel = $this->channelContext->getChannel();
 
         try {
-            $data['price'] = $this->priceCalculator->calculate($object, ['channel' => $this->channelContext->getChannel()]);
-            $data['originalPrice'] = $this->priceCalculator->calculateOriginal($object, ['channel' => $this->channelContext->getChannel()]);
-
+            $data['price'] = $this->priceCalculator->calculate($object, ['channel' => $channel]);
```

Comment:
Tell me under the hood, we are not calculating per se but just retrieving the price for a specific channel?

Reply:
Well... yes 😄